criu/include/util.h

```diff
@@ -383,4 +383,13 @@ static inline void print_stack_trace(pid_t pid) {}
 
 extern int mount_detached_fs(const char *fsname);
 
+#define cleanup_free __attribute__ ((cleanup (cleanup_freep)))
+
+static inline void
```

Comment:
static inline void cleanup_freep (void *p)

you don't need to split this line.

Reply:
done